hack/ci/ci-github-release.sh

```diff
@@ -68,21 +68,21 @@ function upload_archive {
     -H 'Content-Type: application/gzip' \
     -s --data-binary "@${file}")
   if echo "${res}" | jq -e; then
-      # it the response contain errors
-      if echo "${res}" | jq -e '.errors[0]'; then
-        for err in $(echo "${res}" | jq -r '.errors[0].code'); do
-          # if the error code is 'already_exists' do not fail to make this call
-          # idempotent. To make it better we should alse check that the content
-          # match.
-          [[ "${err}" == "already_exists" ]] && return 0
-        done
-         err "Response contains unexpected errors: ${res}"
-        return 1
-      fi
-      return 0
-  else
-      err "Response did not contain valid JSON: ${res}"
+    # it the response contain errors
+    if echo "${res}" | jq -e '.errors[0]'; then
+      for err in $(echo "${res}" | jq -r '.errors[0].code'); do
+        # if the error code is 'already_exists' do not fail to make this call
+        # idempotent. To make it better we should alse check that the content
+        # match.
+        [[ "${err}" == "already_exists" ]] && return 0
+      done
+       err "Response contains unexpected errors: ${res}"
```

Comment:
This line is still indented by one too many spaces ;-)

Reply:
indeed